src/Type/CallableType.php

```diff
@@ -83,4 +83,19 @@ public function isDocumentableNatively(): bool
 		return true;
 	}
 
+	public function isIterable(): int
+	{
+		return self::RESULT_MAYBE;
```

Comment:
Yes, that's one case. The other common is class implementing both `Traversable` & having `__invoke` method. 

But more generally being iterable is entirely independent concept of being callable, so basically callable type = mixed type for all method unrelated to being callable.